diff --git a/target/linux/rockchip/image/armv8.mk b/target/linux/rockchip/image/armv8.mk
index 146af386..67de41b4 100644
--- a/target/linux/rockchip/image/armv8.mk
+++ b/target/linux/rockchip/image/armv8.mk
@@ -676,6 +676,16 @@ $(call Device/seewo_srcm3588-io)
 endef
 TARGET_DEVICES += seewo_srcm3588-sw
 
+define Device/agibot_mb0002-v2
+  DEVICE_VENDOR := AGIBOT
+  DEVICE_MODEL := MB0002 V2
+  SOC := rk3588
+  DEVICE_DTS := rk3588-agibot-mb0002-v2
+  UBOOT_DEVICE_NAME := generic-rk3588
+  IMAGE/sysupgrade.img.gz := boot-common | boot-script | pine64-img | gzip | append-metadata
+endef
+TARGET_DEVICES += agibot_mb0002-v2
+
 define Device/seewo_sv21
   DEVICE_VENDOR := Seewo
   DEVICE_MODEL := SV21
