diff --git a/package/boot/uboot-rockchip/Makefile b/package/boot/uboot-rockchip/Makefile
index fc7cd3d1..4b75551a 100644
--- a/package/boot/uboot-rockchip/Makefile
+++ b/package/boot/uboot-rockchip/Makefile
@@ -430,4 +430,11 @@ define U-Boot/generic-rk3588
     seewo_srcm3588-sw
 endef
 
+define U-Boot/agibot-rk3588
+  $(U-Boot/rk3588/Default)
+  NAME:=AGIBOT MB0002 V2
+  BUILD_DEVICES:= \
+    agibot_mb0002-v2
+endef
+
 define U-Boot/nanopc-t6-rk3588
@@ -521,6 +528,7 @@ UBOOT_TARGETS := \
   generic-rk3576 \
   rock-4d-rk3576 \
   sige5-rk3576 \
+  agibot-rk3588 \
   generic-rk3588 \
   nanopc-t6-rk3588 \
   orangepi-5-plus-rk3588 \
