diff --git a/package/boot/uboot-rockchip/Makefile b/package/boot/uboot-rockchip/Makefile
index fc7cd3d1..616d992f 100644
--- a/package/boot/uboot-rockchip/Makefile
+++ b/package/boot/uboot-rockchip/Makefile
@@ -427,7 +427,8 @@ define U-Boot/generic-rk3588
     radxa_e52c \
     radxa_e54c \
     seewo_srcm3588-io \
-    seewo_srcm3588-sw
+    seewo_srcm3588-sw \
+    agibot_mb0002-v2
 endef
 
 define U-Boot/nanopc-t6-rk3588
