diff --git a/include/target.mk b/include/target.mk
index 1d9e818589..aa26e25bd4 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -58,6 +58,6 @@ DEFAULT_PACKAGES.router:=\
 	iptables-mod-tproxy iptables-mod-extra ipset ip-full default-settings luci luci-proto-ipv6 \
 	ddns-scripts_aliyun ddns-scripts_dnspod luci-app-ddns luci-app-upnp luci-app-autoreboot \
-	luci-app-arpbind luci-app-filetransfer luci-app-vsftpd luci-app-ssr-plus luci-app-vlmcsd \
+	luci-app-arpbind luci-app-filetransfer luci-app-vsftpd luci-app-vlmcsd \
 	luci-app-accesscontrol luci-app-nlbwmon luci-app-turboacc luci-app-wol curl ca-certificates
 
 ifneq ($(DUMP),)
