diff --git a/package/kernel/linux/modules/bluetooth.mk b/package/kernel/linux/modules/bluetooth.mk
index 452de71f2..7bfdf8f4c 100644
--- a/package/kernel/linux/modules/bluetooth.mk
+++ b/package/kernel/linux/modules/bluetooth.mk
@@ -42,11 +42,12 @@ define KernelPackage/hci-uart
   DEPENDS:=+kmod-bluetooth
   KCONFIG:= \
 	CONFIG_BT_HCIUART \
-	CONFIG_BT_HCIUART_BCM=n \
+	CONFIG_BT_HCIUART_BCM=y \
 	CONFIG_BT_HCIUART_INTEL=n \
 	CONFIG_BT_HCIUART_H4 \
 	CONFIG_BT_HCIUART_NOKIA=n
  FILES:= \
-	$(LINUX_DIR)/drivers/bluetooth/hci_uart.ko
+	$(LINUX_DIR)/drivers/bluetooth/hci_uart.ko \
+	$(LINUX_DIR)/drivers/bluetooth/btbcm.ko
  AUTOLOAD:=$(call AutoProbe,hci_uart)
 endef
