diff --git a/target/linux/rockchip/armv8/config-6.12 b/target/linux/rockchip/armv8/config-6.12
--- a/target/linux/rockchip/armv8/config-6.12
+++ b/target/linux/rockchip/armv8/config-6.12
@@ -223,11 +223,11 @@ CONFIG_DEBUG_FS=y
 CONFIG_DEBUG_BUGVERBOSE=y
 CONFIG_DEVFREQ_EVENT_ROCKCHIP_DFI=y
 # CONFIG_DEVFREQ_GOV_PASSIVE is not set
 CONFIG_DEVFREQ_GOV_PERFORMANCE=y
 CONFIG_DEVFREQ_GOV_POWERSAVE=y
 CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=y
 CONFIG_DEVFREQ_GOV_USERSPACE=y
-# CONFIG_DEVFREQ_THERMAL is not set
+CONFIG_DEVFREQ_THERMAL=y
 CONFIG_DEVMEM=y
 # CONFIG_DEVPORT is not set
 CONFIG_DEVTMPFS=y
